spec/unit/message_spec.rb

```diff
@@ -0,0 +1,28 @@
+require 'message'
+require 'pg'
+
+describe Message do
+  describe '.add' do
+    it 'creates and return new message' do
+      user = User.create('test7@example.com', 'password123', 'test7', 'test7')
+      message = Message.add('chit chit', user.id)
+      expect(message.content).to eq 'chit chit'
+      expect(message.username).to eq 'test7'
+    end
+  end
+
+  describe '.all' do
+    it 'call for all messages' do
+      expect(Message.all.first).to be_a Message
```

Comment:
Possibly could do something like Message.all.each here instead of .first, to make sure every item in all is a Message instance?

Reply:
another good point~